lib/netsuite/records/item_vendor.rb

```diff
@@ -0,0 +1,18 @@
+module NetSuite
+  module Records
+    class ItemVendor
+      include Support::Fields
+      include Support::Records
+      include Namespaces::ListAcct
+
+      fields :vendor, :purchase_price, :preferred_vendor
+
+      attr_reader :internal_id
+
+      def initialize(attributes = {})
+        @internal_id = attributes.delete(:internal_id) || attributes.delete(:@internal_id)
```

Comment:
@iloveitaly yeah you're right, the internal_id belongs to the `vendor` not the `item_vendor`. I have fixed it up :)

Thanks